File: Strings/pagina06/3358/3358.java
import java.io.IOException;
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Scanner input = new Scanner(System.in);
        Character[] vogais = {'a', 'e', 'i', 'o', 'u'};
        int n = input.nextInt();

        input.nextLine();

        for (int i = 0; i < n; i++) {
            int count = 0;
            boolean hard = false;
            String sobrenome = input.nextLine();

            for (int j = 0; j < sobrenome.length(); j++) {
                char letra = Character.toLowerCase(sobrenome.charAt(j));

                if (!Arrays.asList(vogais).contains(letra)) {
                    count++;
                } else {
                    count = 0;
                }

                if (count == 3) {
                    hard = true;
                    break;
                }
            }

            if (hard) {
                System.out.println(sobrenome + " nao eh facil");
            } else {
                System.out.println(sobrenome + " eh facil");
            }
        }

        input.close();
    }
}

File: Strings/pagina06/3358/3358.py
n = int(input())
vogais = ['a', 'e', 'i', 'o', 'u']

for _ in range(n):
    sobrenome = input()
    aux = sobrenome.lower()
    hard = False
    count = 0

    for char in aux:
        if char not in vogais:
            count += 1
        else:
            count = 0

        if count == 3:
            hard = True
            break

    if hard:
        print(f"{sobrenome} nao eh facil")
    else:
       print(f"{sobrenome} eh facil") 